android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/webextension/WebExtension.kt

```diff
@@ -470,6 +470,12 @@ data class Metadata(
      * The URL to the detail page of this extension.
      */
     val detailUrl: String?,
+
+    /**
+     * Indicates how the extension works with private browsing windows.
+     * https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/incognito
+     */
+    val incognito: String?,
```

Comment:
As values could only (spanning,split,not_allowed) I think we should [have an enum](https://kotlinlang.org/docs/enum-classes.html), as it will make much easier interacting with the API